### PR TITLE
Return the native promise when it exists

### PR DESCRIPTION
**Pull request by stormtrooper1859**
Fixes https://github.com/sindresorhus/screenfull.js/issues/167

### CODE REVIEW

**Comment by stormtrooper1859** [comment]
@sindresorhus can you review this pr?

**Comment by sindresorhus** [comment]
Did you actually try the code? Because it's erroring when using it with https://github.com/sindresorhus/screenfull.js/blob/master/index.html

File: src/screenfull.js [hunk 1]
```diff
@@ -88,7 +88,11 @@
 
 				element = element || document.documentElement;
 
-				Promise.resolve(element[fn.requestFullscreen]()).catch(reject);
+				var returnPromise = document[fn.requestFullscreen]();
```
Comment (stormtrooper1859) [inline src/screenfull.js]: yeah, there should be `element[fn.requestFullscreen]()`

**Comment by sindresorhus** [comment]
Thanks :)

**Pull request merged**